lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp

```diff
@@ -560,6 +566,13 @@ struct ConvertTritonGPUToLLVM
         id.replaceAllUsesWith(zero);
       });
     }
+    // Update Stats.
+    if (lowerStats.find("memXferBytesWithSMem") != lowerStats.end()) {
+      LLVM_DEBUG(llvm::dbgs()
+                 << "prior value " << memXferBytesWithSMem << '\n');
+      memXferBytesWithSMem += lowerStats["memXferBytesWithSMem"];
```

Comment:
Does this compile?  I cannot find a variable declaration for `memXferBytesWithSmem`.

Reply:
This variable is defined in include/triton/Conversion/TritonGPUToLLVM/Passes.td, it is part of statistics for the lowering pass.